app/jobs/bulk_update_patients_from_pds_job.rb

```diff
@@ -25,7 +25,7 @@ def perform
 
           PatientUpdateFromPDSJob.set(
             priority: 50,
-            wait: 0.21 * index
```

Comment:
I'm starting to wonder if maybe we want this to be configurable outside of the source code. Not sure if we have a good way to do it, though, am I right in thinking that Settings don't get applied unless we deploy a new image or force a redeploy?

Reply:
Yeah that's probably not a bad idea. I think if we had it as an environment variable we'd only need to restart the container I think to get the new change in.